system-int-test/test/cmr/system_int_test/search/service/collection_service_search_test.clj

```diff
@@ -407,7 +407,69 @@
         [coll1 coll2] "service1" {:ignore-case true}
 
         "ignore-case false"
-        [] "service1" {:ignore-case false}))
+        [] "service1" {:ignore-case false})
+
+      ;; update service1's Name. Make sure the collection is reindexed
```

Comment:
This test change is unnecessary. The service update affect collection indexed fields has been tested in other test cases (e.g. `update service affect collection search has-formats field` etc.). If you really want to add a test where the service name is changed during service update. It should be added in a different test case. This test case is for testing different options when searching collections by service name. Adding all these tests for service update is too much noise such that it diminished the goal of this test -- to make sure service info is updated in collection index when service is updated. I am ok with no making any test changes.

Reply:
Removed the test changes